RCAIDE/Library/Plots/Thermal_Management/plot_wavy_channel_conditions.py

```diff
@@ -52,18 +52,18 @@ def plot_wavy_channel_conditions(wavy_channel, results, coolant_line, save_figur
     # get line colors for plots 
     line_colors   = cm.inferno(np.linspace(0,0.9,len(results.segments)))
     
-    fig = plt.figure('Identical_'+ save_filename)
-    fig.set_size_inches(width,height)
-    
-    axis_0 = plt.subplot(1,1,1)
-    axis_1 = plt.subplot(2,2,1)
-    axis_2 = plt.subplot(2,2,2) 
-    axis_3 = plt.subplot(2,2,3)
+    fig_1 = plt.figure(save_filename + '_Coolant_Temp' )
+    fig_2 = plt.figure(save_filename + '_Coolant_Flowrate')
+    fig_3 = plt.figure(save_filename + '_Power')
+    fig_1.set_size_inches(width,height)
+    fig_2.set_size_inches(width,height)
+    fig_3.set_size_inches(width,height)
+     
+    axis_1 = fig_1.add_subplot(1,1,1)
+    axis_2 = fig_2.add_subplot(1,1,1) 
+    axis_3 = fig_3.add_subplot(1,1,1)
     
```

Comment:
Two Many figures will be created if the battery modules are not identical, it is not advisable to do this...

Reply:
Thats because in the regression you are plotting 6x the plots a user would plot. They aren't that bad